example/sample_input.json

```diff
@@ -0,0 +1,10 @@
+{
+    "prompt": "Tell me about the last",
+    "max_tokens": 20,
+    "temperature": 0.75,
+    "k": 0,
+    "p": 0.75,
+    "frequency_penalty": 0,
+    "presence_penalty": 0,
+    "return_likelihoods": "NONE"
```

Comment:
is this right?

Reply:
Fixed to remove return_likelihoods since it is empty if set to None